tests/unit/client.py

```diff
@@ -612,6 +630,27 @@ def test_delete_switch_reserved_chars(self):
         with pytest.raises(BadArgumentError):
             C.switch.delete('nexus/%]-01')
 
+    def test_switch_register(self):
+        """test various cases of switch register"""
+        switchinfo = {
+            "type": "http://schema.massopencloud.org/haas/v0/switches/mock",
```

Comment:
I thought we removed `type` from being in the switchinfo dictionary [here](https://github.com/CCI-MOC/hil/pull/950/files#diff-70caaa8d936ac55788a8e8f85cba7ff3L570)?